src/components/PointsSubscriptionPage.tsx

```
@@ -322,14 +490,92 @@ export default function PointsSubscriptionPage({
-                        <p>포인트 충전 기능이 여기에 들어갑니다.</p>
+                        <div className="grid grid-cols-2 gap-4 mb-4">
+                            {pointOptions.map((option) => (
+                                <div
+                                    key={option.points}
+                                    className={`p-4 border rounded-lg shadow cursor-pointer text-center
+                                    ${
+                                        selectedPointOption?.points === option.points
+                                            ? "border-primary bg-primary-foreground"
+                                            : "border-gray-200 bg-white"
+                                    }`}
+                                    onClick={() => setSelectedPointOption(option)}
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)